scripts/core/INTRO.rst

```diff
@@ -302,3 +302,16 @@ Specific environment variables can be set to control the behavior of unified run
    .. note::
 
     This environment variable should be used together with :envvar:`UR_ENABLE_VALIDATION_LAYER` and :envvar:`UR_LOG_VALIDATION`.
+
+Service identifiers
+---------------------
+
+The Unified Runtime API can log unique device identifiers.
+For security and functionality reasons, the user can enable and disable the logging of identifiers
+using the flags:
+`-DVAL_USE_LIBBACKTRACE_BACKTRACE=ON`,
+`-DUR_ENABLE_TRACING=ON`
```

Comment:
reStructureText inline monospace should be wrapped in double backticks

```suggestion
``-DVAL_USE_LIBBACKTRACE_BACKTRACE=ON``,
``-DUR_ENABLE_TRACING=ON``
```

Reply:
Thank you. Done